src/gsi/qcmod.f90

```diff
@@ -2237,7 +2269,9 @@ subroutine qc_irsnd(nchanl,is,ndat,nsig,ich,sea,land,ice,snow,luse,goessndr,   &
   sfchgtfact=one
   if (zsges > r2000) then
 !    QC1 in statsrad
-     if(luse)aivals(8,is) = aivals(8,is) + one
+     if (.not. cris_sw) then
```

Comment:
Why does the SW changes affect the QC decisions over high topography?

Reply:
It doesn't; this is included so obs aren't counted more than once, since qc_irsnd is called twice in setuprad (calling qc_irsnd twice pertains to the LW vs SW cloud detection... see cloud detection comment)